server/asset/eth/coiner.go

```diff
@@ -66,19 +68,22 @@ func (be *AssetBackend) newSwapCoin(coinID []byte, contractData []byte) (*swapCo
 		return nil, fmt.Errorf("tx %v does not contain initiation with secret hash %x", bc.txHash, bc.secretHash)
 	}
 
+	evmVal := new(big.Int)
 	if be.assetID == BipID {
-		var sum uint64
+		sum := new(big.Int)
 		for _, in := range inits {
-			sum += in.Value
+			sum.Add(sum, in.Value)
 		}
-		if bc.value < sum {
+		if bc.value.Cmp(sum) < 0 {
 			return nil, fmt.Errorf("tx %s value < sum of inits. %d < %d", bc.txHash, bc.value, sum)
 		}
 	}
 
 	return &swapCoin{
 		baseCoin: bc,
 		init:     init,
+		evmVal:   evmVal,
```

Comment:
Is this meant to be used and/or set to something other than 0?

Reply:
Whoops. Leftovers from intermediate iteration.